.github/workflows/wheels.yml

```diff
@@ -132,16 +142,18 @@ jobs:
                             OPENBLAS_NUM_THREADS=2
                             SKLEARN_SKIP_NETWORK_TESTS=1
                             SKLEARN_BUILD_PARALLEL=3
+                            PIP_EXTRA_INDEX_URL=${{ env.PIP_EXTRA_INDEX }}
+                            PIP_PRE=${{ env.PIP_PRE }}
```

Comment:
Continuation of a previous comment: this might be removed here and incorporated in another PR for adapting nightly builds' dependences.